<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>481</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QComboBox" name="lerp_kind_box">
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>线性插值</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三次插值</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>五次插值</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="perlin_kind_box">
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>一维柏林噪声</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>二维柏林噪声</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>三维柏林噪声(未制作)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="perlin_button">
    <property name="text">
     <string>生成结果</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_octave">
      <property name="text">
       <string>Oct</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_oct">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Lacunarity">
      <property name="text">
       <string>Lac</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spin_lac">
      <property name="minimum">
       <double>2.000000000000000</double>
      </property>
      <property name="maximum">
       <double>16.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Persistence">
      <property name="text">
       <string>Per</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="spin_per">
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.050000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_baseX">
      <property name="text">
       <string>baseX</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_baseX">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_baseY">
      <property name="text">
       <string>baseY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_baseY">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_seed">
      <property name="text">
       <string>Seed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spin_seed">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
